specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -530,6 +583,325 @@
+            "schema": {
+              "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Add to reservation": {
+            "$ref": "./examples/AddToReservation.json"
+          },
+          "Remove from reservation": {
+            "$ref": "./examples/RemoveFromReservation.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Browse, reserve and purchase available phone numbers"
+        ],
+        "summary": "Gets a reservation by its ID.",
+        "produces": [
+          "application/json"
+        ],
+        "operationId": "PhoneNumbers_GetReservation",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "reservationId",
+            "description": "The id of the reservation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the reservation.",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumbersReservation"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get reservation": {
+            "$ref": "./examples/GetReservation.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Browse, reserve and purchase available phone numbers"
+        ],
+        "summary": "Deletes a reservation by its ID. Any phone number in the reservation will be released and made available for others to purchase.",
+        "produces": [
+          "application/json"
+        ],
+        "operationId": "PhoneNumbers_DeleteReservation",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "reservationId",
+            "description": "The id of the reservation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Reservation successfully deleted."
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete reservation": {
+            "$ref": "./examples/DeleteReservation.json"
+          }
+        }
+      }
+    },
+    "/availablePhoneNumbers/reservations/{reservationId}/:purchase": {
```

Comment:
Same comment about no slash for a `POST` action unless the API has already shipped that way in the past for other endpoints.